filesystem/keymap.go

```diff
@@ -18,6 +18,8 @@ type KeyMap struct {
 	TempoDown    string     `json:"tempo_down"`
 	AddParam     string     `json:"add_param"`
 	RemoveParam  string     `json:"remove_param"`
+	CopyParams   string     `json:"copy_params"`
+	PasteParams  string     `json:"paste_params"`
```

Comment:
I think `CopyStep` and `PasteStep` would be more appropriate here.

Reply:
Makes sense, thanks!